lib/csv2avro.rb

```diff
@@ -53,8 +67,8 @@ def avro_uri
     "#{dir}/#{name}.avro"
   end
 
-  def error_writer
-    $stderr
+  def log_writer
+    CSV2Avro.log
```

Comment:
It is not necessary to pass `log_writer` around. Just call `CSV2Avro.log` directly everywhere. That makes the code and logging easier to follow.